2021-08-26 07:01:09,408 INFO    MainThread:21945 [wandb_setup.py:_flush():69] setting env: {}
2021-08-26 07:01:09,408 INFO    MainThread:21945 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 07:01:09,408 INFO    MainThread:21945 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_070109-2qcikxjm/logs/debug.log
2021-08-26 07:01:09,408 INFO    MainThread:21945 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_070109-2qcikxjm/logs/debug-internal.log
2021-08-26 07:01:09,410 INFO    MainThread:21945 [wandb_init.py:init():375] calling init triggers
2021-08-26 07:01:09,411 INFO    MainThread:21945 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'epochs': 20, 'batch_size': 32, 'learning_rate': 0.001}
2021-08-26 07:01:09,411 INFO    MainThread:21945 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 2deakazp
2021-08-26 07:01:09,412 INFO    MainThread:21945 [wandb_run.py:finish():1254] finishing run presto105/lv1_p/2deakazp
2021-08-26 07:01:09,562 INFO    MainThread:21945 [jupyter.py:save_history():339] not saving jupyter history
2021-08-26 07:01:09,563 INFO    MainThread:21945 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-08-26 07:01:09,563 INFO    MainThread:21945 [wandb_init.py:_jupyter_teardown():275] cleaning up jupyter logic
2021-08-26 07:01:09,563 INFO    MainThread:21945 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 0
2021-08-26 07:01:09,563 INFO    MainThread:21945 [wandb_run.py:_restore():1526] restore
2021-08-26 07:01:10,566 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 1081
}

2021-08-26 07:01:10,826 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 1081
}

2021-08-26 07:01:11,529 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 10258
}

2021-08-26 07:01:11,632 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 10284
}

2021-08-26 07:01:11,738 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 10284
}

2021-08-26 07:01:11,841 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1587
  total_bytes: 10284
}

2021-08-26 07:01:11,943 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 07:01:12,047 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 07:01:12,149 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 07:01:12,251 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 07:01:12,353 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 07:01:12,778 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 07:01:12,881 INFO    MainThread:21945 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 07:01:14,084 INFO    MainThread:21945 [wandb_run.py:_show_files():1898] logging synced files
2021-08-26 07:01:14,089 INFO    MainThread:21945 [wandb_init.py:init():424] starting backend
2021-08-26 07:01:14,089 INFO    MainThread:21945 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-26 07:01:14,092 INFO    MainThread:21945 [backend.py:ensure_launched():135] starting backend process...
2021-08-26 07:01:14,126 INFO    MainThread:21945 [backend.py:ensure_launched():139] started backend process with pid: 22209
2021-08-26 07:01:14,131 INFO    MainThread:21945 [wandb_init.py:init():429] backend started and connected
2021-08-26 07:01:14,149 INFO    MainThread:21945 [wandb_run.py:_label_probe_notebook():798] probe notebook
2021-08-26 07:01:14,150 INFO    MainThread:21945 [wandb_run.py:_label_probe_notebook():805] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-08-26 07:01:14,150 INFO    MainThread:21945 [wandb_init.py:init():477] updated telemetry
2021-08-26 07:01:14,151 INFO    MainThread:21945 [wandb_init.py:init():500] communicating current version
2021-08-26 07:01:14,970 INFO    MainThread:21945 [wandb_init.py:init():505] got version response 
2021-08-26 07:01:14,970 INFO    MainThread:21945 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-26 07:01:15,198 INFO    MainThread:21945 [wandb_init.py:init():540] starting run threads in backend
2021-08-26 07:01:19,750 INFO    MainThread:21945 [wandb_run.py:_console_start():1584] atexit reg
2021-08-26 07:01:19,751 INFO    MainThread:21945 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.WRAP
2021-08-26 07:01:19,751 INFO    MainThread:21945 [wandb_run.py:_redirect():1495] Wrapping output streams.
2021-08-26 07:01:19,752 INFO    MainThread:21945 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-26 07:01:19,752 INFO    MainThread:21945 [wandb_init.py:init():565] run started, returning control to user process
